test/datatypes/interval.coffee

```diff
@@ -68,15 +68,15 @@ describe 'DateTimeInterval.contains', ->
 
   it 'should properly handle imprecision', ->
     @all2012.closed.contains(@bef2012.toMonth).should.be.false()
-    @all2012.closed.contains(@beg2012.toMonth).should.be.true()
+    should.not.exist @all2012.closed.contains(@beg2012.toMonth)
```

Comment:
Why?  Shouldn't this be true?

Reply:
Oh crap.  It's because of the way the spec defines the calculation for `same or before` and `same or after` now, isn't it?  Well... that sucks. (sigh).

Reply:
I see this affected a lot of things below.  I won't whine at every single one.  It's not your fault.